cf-serverd/server_tls.c

```diff
@@ -429,7 +429,7 @@ int ServerTLSSessionEstablish(ServerConnectionState *conn)
         int ret = SSL_accept(ssl);
         if (ret <= 0)
         {
-            TLSLogError(ssl, LOG_LEVEL_ERR,
+            (void)TLSLogError(ssl, LOG_LEVEL_ERR,
```

Comment:
No errors. Just good practice. IMHO it's never to late to start doing something better. And the good reason is that you are explicitly mentioning in the code that you are ignoring the result of function which is really useful for future maintenance.